R/CallbackSetTB.R

```diff
@@ -0,0 +1,85 @@
+#' @title TensorBoard Logging Callback
+#'
+#' @name mlr_callback_set.tb
+#'
+#' @description
+#' Logs training loss, training measures, and validation measures as events.
+#' To view them, use TensorBoard with `tensorflow::tensorboard()` (requires `tensorflow`) or the CLI.
+#' @details
+#' Logs events at most every epoch.
+#'
+#' @param path (`character(1)`)\cr
+#'   The path to a folder where the events are logged.
+#'   Point TensorBoard to this folder to view them.
+#' @param log_train_loss (`logical(1)`)\cr
+#'  Whether we log the training loss.
+#' @family Callback
+#' @export
+#' @include CallbackSet.R
+CallbackSetTB = R6Class("CallbackSetTB",
+  inherit = CallbackSet,
+  lock_objects = FALSE,
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function(path, log_train_loss) {
+      self$path = assert_path_for_output(path)
+      if (!dir.exists(path)) {
+        dir.create(path, recursive = TRUE)
+      }
+      self$log_train_loss = assert_logical(log_train_loss)
+    },
+    #' @description
+    #' Logs the training loss, training measures, and validation measures as TensorFlow events.
+    on_epoch_end = function() {
+      if (self$log_train_loss) {
+        private$.log_train_loss()
+      }
+
+      if (length(self$ctx$last_scores_train)) {
+        map(names(self$ctx$measures_train), private$.log_train_score)
```

Comment:
```suggestion
        walk(names(self$ctx$measures_train), private$.log_train_score)
```

Reply:
`map` returns something but here we only call into the function for side-effects -> use `walk()`

Reply:
(which returns `NULL`)